packages/client/src/SubmissionController.test.ts

```diff
@@ -20,14 +20,16 @@ import {
   REGISTER_BIRTH_APPLICATION,
   REJECT_BIRTH_APPLICATION,
   COLLECT_BIRTH_CERTIFICATE,
-  ARCHIVE_BIRTH_DECLARATION
+  ARCHIVE_BIRTH_DECLARATION,
+  REINSTATE_BIRTH_APPLICATION
 } from '@client/views/DataProvider/birth/mutations'
 import {
   ARCHIVE_DEATH_DECLARATION,
   APPROVE_DEATH_APPLICATION,
   COLLECT_DEATH_CERTIFICATE,
   REGISTER_DEATH_APPLICATION,
-  REJECT_DEATH_APPLICATION
+  REJECT_DEATH_APPLICATION,
+  REINSTATE_DEATH_APPLICATION
```

Comment:
We need to refactor `REINSTATE_DEATH_APPLICATION` to `REINSTATE_DEATH_DECLARATION`

Reply:
Changed